docs/integration.rst

```diff
@@ -1113,22 +1113,27 @@ To enable building RAUC bundles, set::
 in your platformconfig (by using ``ptxdist platformconfig``).
 
 This adds a default image recipe for building a RAUC update bundle out of the
-system's rootfs. As for all image recipes, the `genimage` tool is used to
-configure and generate the update bundle.
+system's rootfs.
+As for most image recipes, the `genimage <genimage_>` tool is used to configure
+and generate the update bundle.
 
 PTXdist's default bundle configuration is placed in
-`config/images/rauc.config`. You may also copy this to your platform directory
-to use this as a base for custom bundle configuration.
+`config/images/rauc.config`.
+You may also copy this to your platform directory to use this as a base for
+custom bundle configuration.
 
-In order to sign your update (mandatory) you also need to place a valid
-certificate and key file in your BSP at the following paths::
-
-  $(PTXDIST_PLATFORMCONFIGDIR)/config/rauc/rauc.key.pem (key)
-  $(PTXDIST_PLATFORMCONFIGDIR)/config/rauc/rauc.cert.pem (cert)
+RAUC enforces signing of update bundles.
+PTXdist versions since 2020.06.0 use its `code signing infrastructure
+<ptxdist-code-signing_>`_ for signing and keyring verification.
+Previous versions expected the signing key in
+``$(PTXDIST_PLATFORMCONFIGDIR)/config/rauc/rauc.key.pem``.
```

Comment:
It sounds a bit redundant to the above 'note' paragraph, but I also do not have a really better idea of how to change this.

Reply:
True. I tend to leave it this way, so everyone is aware of the changes, for key and keyring.